.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM zmkfirmware/zmk-dev-arm:2.4
+FROM docker.io/zmkfirmware/zmk-dev-arm:2.5-branch-20210617193230-2.5.0-0.12.4-5d9cfa5ab7a8-947598130
```

Comment:
Careful of the `docker.io/` when you set it to `2.5` before the merge.

`.devcontainer`, `build.yml` and `test.yml` should be identical.

Reply:
Does this negatively impact regular docker usage? This is helpful on Fedora with podman, since it has several registries as options configured out of the box.

Not needed for the GH Actions, but a change I'd like to add here if we can. I can defer that post 2.5 of course.

Reply:
> Does this negatively impact regular docker usage?

No, it's just conventional.

I'm happy with `docker.io/` as long as it's consistent everywhere - including `zmk-docker`.

Let's follow it up immediately after 2.5.

Reply:
WFM. I've reverted this part for now.